ExampleSwift/DigiMeSDKExampleSwift.xcodeproj/project.pbxproj

```diff
@@ -39,6 +40,7 @@
 
 /* Begin PBXFileReference section */
 		06BD81C8203F1B0100A89001 /* CA_RSA_PRIVATE_KEY.p12 */ = {isa = PBXFileReference; lastKnownFileType = file; path = CA_RSA_PRIVATE_KEY.p12; sourceTree = "<group>"; };
+		1E831EE92045D50D008F0650 /* StoreKit.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = StoreKit.framework; path = System/Library/Frameworks/StoreKit.framework; sourceTree = SDKROOT; };
```

Comment:
We should remove the StoreKit framework from the example apps, and instead add it to the pod itself, right? I'm guessing via updating the podspec.